pipeline-runner/R/gem2s-2-load_cellranger_files.R

```diff
@@ -9,14 +9,14 @@
 #' }
 #' @export
 #'
-load_cellranger_files <- function(input, pipeline_config, prev_out) {
+load_cellranger_files <- function(input, pipeline_config, prev_out, input_dir = '/input') {
```

Comment:
Why do we use this as a parameter? We are not getting this in this function, right?

Reply:
And if we were why wouldn't we send it inside input?

Reply:
This is used to allow running tests locally. `/input` doesn't exist locally so I made it a parameter so that the function can be tested with a temporary directory.